### ISSUE TITLE
Feature request: Use the data from Beszel in Home Assistant

### ISSUE THREAD

**Issue by jtenniswood** [henrygd/beszel#356]
The is probably WAY out of scope, but I would love to be able to use the data collected in Beszel inside home assistant (for at a glance dashboards). I use Glances for this today, but would love to replace it with Beszel.

If anyone has any time, any help is appreciated!

**Comment by henrygd** [comment]
I agree, however I have no idea how that works. I'll definitely look into it when I have time.

**Comment by eric72777** [comment]
Some reference info in case someone is interested in creating an integration:

[Creating your first integration](https://developers.home-assistant.io/docs/creating_component_index/)

[glances integration](https://github.com/home-assistant/core/tree/dev/homeassistant/components/glances)

**Comment by maxexcloo** [comment]
Have vibe coded an integration, seems to work okay: https://github.com/maxexcloo/beszel-homeassistant-integration

**Comment by cgfm** [comment]
Same here. I've another one that supports less system stats but creates also devices for docker containers if this is helpful for you.

https://github.com/cgfm/Beszel-HASS

**Comment by svenvg93** [comment]
Closing this one as there are community add-ons available and listed in the docs.
